ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -133,14 +133,11 @@ Control {
         SequentialAnimation {
             id: messageFoundAnimation
 
-            PauseAnimation {
-                duration: 600
-            }
             NumberAnimation {
                 target: highlightRect
                 property: "opacity"
                 to: 1.0
-                duration: 1500
+                duration: 250
```

Comment:
Does that reflect the design?

Reply:
I don't think we have designs for this. 
The old animation was driving me crazy during work on this PR 😄 

Probably we could ask designers to create strict animation rules for this, but such fast change is very easy to make ourselves.

@benjthayer, do you think this change make sense?

### The old animation is extremely slow:

https://github.com/status-im/status-desktop/assets/25482501/9c8b46fb-704e-47e0-8574-31acf5484169

### The new on is much closer to Discord:

https://github.com/status-im/status-desktop/assets/25482501/e87f42c5-e747-4ad4-8c36-06f0e92a79c6

### Discord:

https://github.com/status-im/status-desktop/assets/25482501/87624901-3513-4c78-b168-9e3ebeb0f08a

Reply:
Ok yeah I agree, the old one was very slow. I don't mind keeping the new one. Designers can create a ticket for the value they want (Ben is on vacation I think)

Reply:
> I don't think we have designs for this. The old animation was driving me crazy during work on this PR 😄
> 
> Probably we could ask designers to create strict animation rules for this, but such fast change is very easy to make ourselves.
> 
> @benjthayer, do you think this change make sense?
> 
> ### The old animation is extremely slow:
>  Screen.Recording.2023-08-04.at.12.08.35.mov 
> ### The new on is much closer to Discord:
>  Screen.Recording.2023-08-04.at.12.13.08.mov 
> ### Discord:
>  Screen.Recording.2023-08-04.at.12.09.40.mov

Looks great to me thanks @igor-sirotin , nice enhancement 👌 . We can always tweak further / define values if needed  once we've had a chance to test. Thanks